lib/alephant/renderer/views/html.rb

```diff
@@ -1,64 +1,57 @@
-require "alephant/renderer/views/base"
-require "mustache"
-require "i18n"
+require 'alephant/renderer/views/base'
+require 'alephant/renderer/engine/mustache'
+require 'alephant/renderer/i18n/locale_component_yaml'
 
 module Alephant
   module Renderer
     module Views
-      class Html < Mustache
+      class Html
         include ::Alephant::Renderer::Views::Base
 
+        class << self
+          # FIXME: remove this when we've gotten rid of all of the `self.template_path` directives
```

Comment:
We don't have to actually do that in the old codebases.  Just not use it in the new ones - this was just to supply a little bit of backwards compatibility (i.e. not make things break if someone does want to do the update)